### PR TITLE
Add addprop command

### PR DESCRIPTION
**Pull request by zsiggg**
Work towards #69

* Syntax: addprop -n <name> -price <price> -a <address> -d <description> [-t <tag>]...
    * There can be multiple tags
* Currently only prints out a success message when command is entered correctly, without adding to a model or saving to storage

Todo:
* Model for Properties
* Storage for Properties

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddPropertyCommand.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            + PREFIX_PRICE + " 500000 "
+            + PREFIX_ADDRESS + " 311, Clementi Ave 2, #02-25 "
+            + PREFIX_DESCRIPTION + " ready to be bought "
+            + PREFIX_TAG + " condo "
+            + PREFIX_TAG + " low floor ";
+
+    public static final String MESSAGE_SUCCESS = "New property added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PROPERTY = "This property already exists in the address book";
+
+    private final Property toAdd;
```
Comment (riccqi) [inline src/main/java/seedu/address/logic/commands/AddPropertyCommand.java:39]: clearer to rename to `property`?

File: src/main/java/seedu/address/model/property/PropertyAddress.java [hunk 2]
```diff
@@ -5,9 +5,9 @@
 
 /**
  * Represents a Property's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidPropertyAddress(String)}
  */
-public class Address {
+public class PropertyAddress {
```
Comment (riccqi) [inline src/main/java/seedu/address/model/property/PropertyAddress.java:10]: We can probably move the Address class out into the model folder, so both Person and Property can share the same address validation class

**Review by riccqi — APPROVED** [review]
lgtm, some nits

File: src/main/java/seedu/address/model/property/PropertyAddress.java (hunk 2 above)
Comment (zsiggg) [reply src/main/java/seedu/address/model/property/PropertyAddress.java]: resolved in latest commit [3c58a46](https://github.com/AY2223S1-CS2103T-F12-1/tp/pull/70/commits/3c58a46dac9abb082c1ce3bf0c16f5c59004866a)

**Pull request merged**